### DISCUSSION TITLE
International backend hosting locations

### DISCUSSION THREAD

**Discussion by steveninety** [swellstores/community#72] [Ideas]
Am I correct that the Swell backend is hosted only in the US? 
Is there any chance more hosting locations will be added to the list in the near future? That would speed up API requests for the non-US users :)

**Comment by ericingram** [comment]
We are looking forward to adding more backend API locations outside the US. No exact timeline yet but it is likely this year.

**Reply by genox** [reply]
I would like to emphasise this. I am located in Central Europe and the admin ui feels incredibly sluggish at times. If this is indicative of the expected API performance, it is a bit disappointing.

My client has to add around 200 products, I can already feel his frustration before he started, with saves and route changes taking anything from 250ms to 2.5s response time. Not a stellar first impression. I hope you can provide EU hosting soon.
